test/kitchen/tasks/kitchen.py

```diff
@@ -37,7 +39,24 @@ def genconfig(
     if not platlist and not provider:
         provider = "azure"
 
-    platforms = load_platforms(ctx, platformfile=platformfile)
+    if platformfile:
+        with open(platformfile, "r") as f:
+            platforms = json.load(f)
+    else:
+        try:
+            print(
+                "Fetching the latest kitchen platforms.json from Github. Use --platformfile=platforms.json to override with a local file."
+            )
+            r = requests.get(
+                'https://raw.githubusercontent.com/DataDog/datadog-agent/master/test/kitchen/platforms.json',
+                allow_redirects=True,
+            )
+            platforms = json.loads(r.content)
```

Comment:
Actually, I didn't remember the requests library had a builtin json loader 😅 we can do the following
```suggestion
            r = requests.get(
                'https://raw.githubusercontent.com/DataDog/datadog-agent/master/test/kitchen/platforms.json',
                allow_redirects=True,
            )
            r.raise_for_status()
            platforms = r.json()
```

Also added the `raise_for_status` which will make the exception more readable